mesonbuild/dependencies/configtool.py

```diff
@@ -31,13 +31,16 @@ class ConfigToolDependency(ExternalDependency):
     Takes the following extra keys in kwargs that it uses internally:
     :tools List[str]: A list of tool names to use
     :version_arg str: The argument to pass to the tool to get it's version
+    :skip_version str: The argument to pass to the tool to ignore its version
+        Because some tools are stupid and don't accept --version
```

Comment:
I think something like
```
:fallback_version_arg str: The argument to pass to the tool to get it's version if calling it with ``version_arg`` fails. Because some tools are stupid and don't accept --version.
```
better describes what the code does.

Reply:
```diff
     :skip_version str: The argument to pass to the tool to ignore its version
+        (if ``version_arg`` fails, but it may start accepting it in the future)
         Because some tools are stupid and don't accept --version
```